projects/v2/src/app/models/ga.model.ts

```diff
@@ -0,0 +1,40 @@
+// Standard set of actions for GA events
+export enum GaAction {
+  // Action for form inputs
+  INPUT = 'input',
+  // Action for toggle ui elements
+  TOGGLE = 'toggle',
+  // Action for slider change enents
+  SLIDE = 'slide',
```

Comment:
I haven't seen, but is this event fired everytime the slide is being used or is it fired when the slide action has been completed?

Reply:
They are only fired when the slide action is completed.